src/screens/GroupInfo/elements/ModalAction.js

```diff
@@ -87,6 +87,16 @@ const ModalAction = (props) => {
                 </View>
               </TouchableOpacity>
             )}
+            {isGroup && (
+              <TouchableOpacity
+                onPress={() => onPress('remove', selectedUser)}
+                style={styles.buttonStyle}>
+                <View style={{flexDirection: 'row'}}>
+                  <Text style={styles.textButton}>Remove {name}</Text>
+                  <Loading visible={isLoadingInitChat} />
+                </View>
+              </TouchableOpacity>
+            )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code block introduces a "Remove" button for group chats. However, there should be some checks to ensure that the current user has the necessary permissions to remove other users from the group. Without these checks, any member could potentially remove others, which could lead to misuse or errors.

```diff
            {isGroup && currentUserHasRemovePermission && (
              <TouchableOpacity
                onPress={() => onPress('remove', selectedUser)}
                style={styles.buttonStyle}>
                <View style={{flexDirection: 'row'}}>
                  <Text style={styles.textButton}>Remove {name}</Text>
                  <Loading visible={isLoadingInitChat} />
                </View>
              </TouchableOpacity>
            )}
```
In the above diff, `currentUserHasRemovePermission` is a hypothetical variable that you would need to implement. It should represent whether the current user has the necessary permissions to remove other users from the group.


<!-- This is an auto-generated comment by OSS CodeRabbit -->